R/calcBiomassPrices.R

```diff
@@ -9,6 +9,24 @@ calcBiomassPrices <- function(){
   x <- readSource("MAgPIE", subtype = "supplyCurve_magpie_40")
```

Comment:
I would expect here a different subtype (e.g. subtype = "supplyCurve_magpie_4p4"

Reply:
Hi Lavinia, we started discussing this in the RSE vault channel yesterday but did not finish. Since I wanted to make the new emulators available as soon as possible, I approved the PR before we decided on how to deal with the subtype. So could you approve please and we will find a better solution soon for the subtype. Thanks.